mods/sgmnm/formats-data.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+exports.BattleFormatsData = {
+	aegislash: {
+		inherit: true,
+		tier: "OU",
+	},
+	blaziken: {
+		inherit: true,
+		tier: "OU",
+	},
+	deoxysdefense: {
+		inherit: true,
+		tier: "OU",
+	},
+	deoxysspeed: {
+		inherit: true,
+		tier: "OU",
+	},
+	genesect: {
+		inherit: true,
+		tier: "OU",
+	},
+	landorus: {
+		inherit: true,
+		tier: "OU",
+	},
+	marshadow: {
+		inherit: true,
+		tier: "OU",
+	},
+	slaking: {

```

Comment:
Why don't you just add Slaking to the banlist (also would Archeops be broken?)

Reply:
its UBER meaning it cannot mega evolve, and no, If mix and mega didnt ban archeops than why should i?

Reply:
its UBER meaning it cannot mega evolve, and no, If mix and mega didnt ban archeops than why should i?

Reply:
Mk